### PR TITLE
Feature/update version to 0 6.6[PARA CERRAR]

### PR DESCRIPTION
**Pull request by Jiang-dxc**
Pendiente de revisar [declaración de owner en contratos heredados duplicado](https://github.com/alastria/alastria-identity/compare/feature/update_version_to_0_6.6?expand=1#diff-aead9617d6ddc3d3a73c22e7633c7049R6), si es un typo o es intencionado.

### CODE REVIEW

**Comment by kkx** [comment]
Repetido, cerrarlo please.

File: contracts/identityManager/AlastriaProxy.sol [hunk 1]
```diff
@@ -1,23 +1,25 @@
-pragma solidity 0.4.23;
+pragma solidity 0.5.17;
 
 import "../libs/Owned.sol";
 
 contract AlastriaProxy is Owned {
-    address public owner;
+    //address public owner;
```
Comment (nefera606) [inline contracts/identityManager/AlastriaProxy.sol:6]: Remove, is not used

File: contracts/identityManager/AlastriaProxy.sol [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
 
     event Forwarded (address indexed destination, uint value, bytes data);
 
     //TODO: upgradeable owner for version in Identity Manager
-    constructor () public {
-        owner = msg.sender;
-    }
+    //constructor () public {
```
Comment (nefera606) [inline contracts/identityManager/AlastriaProxy.sol:11]: Same as above

File: contracts/identityManager/AlastriaProxy.sol (hunk 1 above)
Comment (Jiang-dxc) [reply contracts/identityManager/AlastriaProxy.sol:6]: Este merge request esta repetido, es para cerrarlo, que no tengo acceso para cerrarlo, cierralo please.

File: contracts/misc/Migrations.sol [hunk 3]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.23;
+pragma solidity ^0.5.17;
```
Comment (nefera606) [inline contracts/misc/Migrations.sol:1]: If we are using a fixed version in the other contracts, can it be fixed in migrations?

**Comment by nefera606** [comment]
Cierro la request por ser similar a #86 Feature/add openzeppelin upgrade

**Pull request closed without merging**